steering_controllers_library/test/test_steering_controllers_library.hpp

```diff
@@ -172,22 +170,22 @@ class SteeringControllersLibraryFixture : public ::testing::Test
     command_ifs.reserve(joint_command_values_.size());
 
     command_itfs_.emplace_back(hardware_interface::CommandInterface(
-      rear_wheels_names_[0], traction_interface_name_,
+      traction_joints_names_[0], traction_interface_name_,
```

Comment:
Here it is always assumed that there are exactly two joints in the order [right, left]. This is as far as I can tell not documented anywhere.

Reply:
only for ackermann, and there is this note in the beginning of the test:
```
// NOTE: Testing steering_controllers_library for Ackermann vehicle configuration only
```
I added this information to the parameter yaml.